frontends/election-manager/src/utils/generate_pdf_export_metadata_csv.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  BallotLocales,
+  ElectionDefinition,
+  getPrecinctById,
+} from '@votingworks/types';
+import {
+  getBallotPath,
+  getBallotStylesData,
+  sortBallotStyleDataByPrecinct,
+} from './election';
+import { BallotMode } from '../config/types';
+
+/**
+ *
+ * Creates a CSV metadata file for ballot PDF export zip archive. Includes a
+ * row for each PDF file included in the zip archive.
+ * @param electionDefinition Contains the election schema and hash
+ * @param ballotMode Identifies the export as for official, test, or sample ballots
+ * @param isAbsentee Identifies the export as for absentee or precinct ballots
+ * @param ballotLocales Identifies the export as for a given BallotLocales
+ * @returns string file content for a CSV file with export metadata
+ */
+
+const headers = ['Filename', 'Precinct', 'Ballot Style'];
+
+export function generatePdfExportMetadataCsv({
+  electionDefinition,
+  ballotMode,
+  isAbsentee,
+  ballotLocales,
+}: {
+  electionDefinition: ElectionDefinition;
+  ballotMode: BallotMode;
+  isAbsentee: boolean;
+  ballotLocales: BallotLocales;
+}): string {
+  const { election, electionHash } = electionDefinition;
+  const ballotStyleList = sortBallotStyleDataByPrecinct(
+    election,
+    getBallotStylesData(election)
+  );
+
+  let finalDataString = headers.join(', ');
```

Comment:
I believe CSVs aren't expected to have spaces after the commas that separate entries ([relevant link](https://discuss.codecademy.com/t/in-a-csv-file-does-spacing-matter-before-and-after-commas/353056))